packages/lesswrong/components/posts/PostsPage.jsx

```diff
@@ -252,6 +241,10 @@ class PostsPage extends Component {
       const view = _.clone(router.location.query).view || Comments.getDefaultView(post, currentUser)
       const description = post.plaintextExcerpt ? post.plaintextExcerpt : (post.body && post.body.substring(0, 300))
       const commentTerms = _.isEmpty(query && query.view) ? {view: view, limit: 500} : {...query, limit:500}
+      
+      const sectionData = post.tableOfContents;
+      const htmlWithAnchors = sectionData ? sectionData.html : post.htmlBody;
```

Comment:
I don't like this data structure. I think I would prefer this thing to be split out into two fields, it that is at all possible (though it might not be, because of the way resolvers work)